src/Onebip/DateTime/UTCDateTime.php

```diff
@@ -423,6 +423,27 @@ public function startOfMonth()
         );
     }
 
+    public function endOfMonth()
+    {
+	return self::box(
+	    $this->toDateTime()->format('Y-m-t')
+	);
+    }
+
+    public function startOfWeek()
+    {
+	return self::box(
+	    $this->toDateTime()->format('monday this week')
+	);
+    }
+
+    public function endOfWeek()
```

Comment:
not for all European people the week end with `sunday`, be more explicit in the name of the function